CSharpHacks/CSharpHacks.Tests/DoubleTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using FluentAssertions;
+using Xunit;
+
+namespace CSharpHacks.Tests
+{
+    public class DoubleTests
+    {
+        [Fact]
+        public void Zero_should_be_zero()
+        {
+            double value = 0;
+            value.IsZero().Should().Be(true);
```

Comment:
this is fine, but there's actually a "BeZero()" function available

Reply:
How would you use it?

In this case I'm testing the method `IsZero()`, which returns bool, so assertion should be comparing bools surely?

Reply:
now I think about it, I'm less sure.  In any case, Fluent Assertions is only for use in unit tests.  Much as we might sometimes like to, we can't use their features in production code